linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml

```diff
@@ -1,14 +1,21 @@
 documentation_complete: true
```

Comment:
I think these `accounts_tmout` can be removed as per: https://github.com/ComplianceAsCode/content/pull/6631#discussion_r584669502